app/api/templates.v2/routes/validators/createNewRelationshipProperty.ts

```diff
@@ -14,6 +14,7 @@ const createNewRelationshipPropertySchema: ValidatorSchema<
   optionalProperties: {
     _id: { type: 'string' },
     denormalizedProperty: { type: 'string' },
+    targetTemplates: { elements: { type: 'string' } },
```

Comment:
I think this is likely the reason for the bug you found. I will take a look at this. For the record, in the ideal world we don't even accept target templates here, because it's a calculated property. But for now we have no option given the v1-v2 integration.